### ISSUE TITLE
number cluster names from clusterSingle don't match clusterIds

### ISSUE THREAD

**Issue by epurdom** [epurdom/clusterExperiment#227]
Not a huge issue, but a bit annoying that the numeric clustering indices from clusterSingle don't match the given internal clusterIds that are assigned. Should check out because might be symptom of something strange in either clusterSingle or internal assignment.

Example from tests:

<pre>
  cl1 <- clusterSingle(smSimData, 
                       subsample=FALSE, sequential=FALSE,
                       mainClusterArgs=list(clusterFunction="pam",clusterArgs=list(k=6)),isCount=FALSE)
clusterLegend(cl1)
</pre>

Gives

<pre>
$clusterSingle
  clusterIds color     name
1 "1"        "#1F78B4" "4" 
2 "2"        "#33A02C" "1" 
3 "3"        "#E31A1C" "5" 
4 "4"        "#FF7F00" "2" 
5 "5"        "#6A3D9A" "3" 
6 "6"        "#B15928" "6" 
</pre>

**Comment by epurdom** [comment]
Simpler example (now in tests) which fails:
```
  set.seed(151789)
  charValue<-sample(rep(c("-1","A","B","C","D"),times=c(2,3,2,1,7)))
  newCC<-addClusterings(ccSE,charValue,clusterLabel="characterCluster")
  expect_equal(tableClusters(newCC,"characterCluster",useNames=TRUE),table(charValue))
```

**Comment by epurdom** [comment]
Fixed in 2.1.2-9008 (added `sort` into `.makeIntegerClusters`)
